app/controllers/concerns/encryptables.rb

```diff
@@ -56,6 +56,49 @@ def credential_id
     params[:credential_id]
   end
 
+  ### Sharing ###
```

Comment:
since we're using domain class EncryptableTransfer ... it probably does not require an additional file encryptable concern. 
so I would suggest removing this completely and add some minimal code to EncryptablesContrroller directly